server/mlabns/util/maxmind.py

```diff
@@ -29,16 +30,24 @@ def __ne__(self, other):
         return not self.__eq__(other)
 
 
-def get_ip_geolocation(ip_address):
+def get_ip_geolocation(ip_address, address_family):
     """Returns the geolocation data associated with an IP address from MaxMind.
 
     Args:
         ip_address: A string describing an IP address (v4 or v6).
+        address_family: the ip_address format, either ipv4 or ipv6
 
     Returns:
         A populated GeoRecord if matching geolocation data is found for the
         IP address. Otherwise, an empty GeoRecord.
     """
+    maxmind_file = None
+    if address_family == message.ADDRESS_FAMILY_IPv4:
+        maxmind_file = constants.GEOLOCATION_MAXMIND_CITY_FILE_IPv4
+    elif address_family == message.ADDRESS_FAMILY_IPv6:
+        maxmind_file = constants.GEOLOCATION_MAXMIND_CITY_FILE_IPv6
+
```

Comment:
Well I wasn't sure. The ip should be properly formatted because that's checked when the LookupQuery object is created. My thought process for not having an else is just that I don't like sneaky errors, if it's not set to ipv4 or ipv6 here it should remain None which would be a more obvious error. 

I could be persuaded to change to if/else though if you think it's better.